### PR TITLE
[READY] [ANTAG] IT CAME FROM OUTER SPACE....

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

WIP

* Run around maint competing with lizards for rats
* Try not to die

## Why It's Good For The Game

Not ready yet, just for testing!

## Changelog
:cl: Kmc2000 & Azlan
add: Bloodling! An evil space-creature that wants to eat the ship's pets. 
/:cl:

### CODE REVIEW

**Comment by Bokkiewokkie** [comment]
What about AI shell functionality n stuff

**Comment by RuneQuester** [comment]
Ah yes,Carrion

**Comment by Karmic-Skink** [comment]
> Ah yes,Carrion

Not Carrion.

**Comment by Pontenerd** [comment]
Mmmm, new badman

**Comment by Kmc2000** [comment]
Balance shit:
Gimp bloodling clings - remove most abilities, ESP. revive. Must be healed by the master
Give EMP screech to bloodling, remove from cling (MAYBE: give it a flash screech instead? Softer stun that way.)
Increase infest cling cost
Increase bloodling low tier hitbox
Bloodling absorb drops cloak, and CDs it.
Can only transfer maxBio - 10 biomass to avoid instadying

Consider:
Bloodling can give biomass to his clings to give them unlock points

Review:
Win condition. Waiting for 10 mins on a worm rampage kinda sucks ass

Fix:
WORM STUCK 
Bugged minion spawns (runtime in infest ... etc)

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 /obj/machinery/door/CanPass(atom/movable/mover, turf/target)
 	if(istype(mover) && (mover.pass_flags & PASSGLASS))
 		return !opacity
-	return !density
+	return (mover.pass_flags & PASSDOOR) || !density
```
Comment (Bokkiewokkie) [inline code/game/machinery/doors/door.dm:136]: Add //NSV13

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
You should probably make sure to check for any squads related issues and mention them as fixed if this PR fixes them. (#691 for example)

File: code/game/machinery/doors/door.dm [hunk 2]
```diff
@@ -134,7 +134,7 @@
...
```
Comment (Kmc2000) [inline code/game/machinery/doors/door.dm:137]:
```suggestion
	return (mover.pass_flags & PASSDOOR) || !density //nsv13 - bloodling passdoor
```

**Comment by DrSirKnight** [comment]
Becoming a headspider breaks a lot of things. You can still speak in changeling hivemind chat (With button and with prefix) if you're a headspider but you can't hear them and possibly nobody can hear you aswell. After getting revived you can't use points to purchase anything even if you have enough points left.

**Comment by Kmc2000** [comment]
-Headcrabs, investigate
-Nerf slam 
-Mindshield conversions are loud

**Comment by Bapldap** [comment]
Last night when I was playing it I noticed that the ascended worm is basically on 'help' intent which causes some weird movements when you run over someone or a table. Also my health was constantly at a low state before I ascended, displaying 1% though I got it up 13% when I gathered biomass. Good antag, I found the stages before ascension more fun, it didn't seem like flesh path ascension fitted well with it.

**Pull request merged**